<ns0:test-suite xmlns:ns0="urn:model.allure.qatools.yandex.ru" start="1552015193905" stop="1552015194025">
  <name>scripts.test_login</name>
  <labels/>
  <test-cases>
    <test-case start="1552015193905" status="passed" stop="1552015193975">
      <name>TestLogin.test_login1</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="13068-MainThread"/>
        <label name="host" value="DESKTOP-3SHNI8B"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1552015193975" status="failed" stop="1552015194025">
      <name>TestLogin.test_login2</name>
      <failure>
        <message>AssertionError: assert 0</message>
        <stack-trace>self = &lt;scripts.test_login.TestLogin object at 0x000001B1690F3B38&gt;

    def test_login2(self):
&gt;       assert 0
E       assert 0

scripts\test_login.py:6: AssertionError</stack-trace>
      </failure>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="13068-MainThread"/>
        <label name="host" value="DESKTOP-3SHNI8B"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
    <test-case start="1552015194025" status="passed" stop="1552015194025">
      <name>TestLogin.test_login3</name>
      <attachments/>
      <labels>
        <label name="severity" value="normal"/>
        <label name="thread" value="13068-MainThread"/>
        <label name="host" value="DESKTOP-3SHNI8B"/>
        <label name="framework" value="pytest"/>
        <label name="language" value="cpython3"/>
      </labels>
      <steps/>
    </test-case>
  </test-cases>
</ns0:test-suite>
